nep-Y.mediawiki

```diff
@@ -108,7 +108,7 @@ Parameter object has the following structure:
 
 <code>type</code> indicates the type of the parameter. It can have any value of the ParameterType enumeration defined below, except for <code>Void</code>.
 
-Parameter may also have all of the ExtendedType fields embedded in the same
+Parameter may also have all of the ExtendedType fields embedded /* Why are these fields embedded? For Method we have a separate `extendedreturntype` field, I think it will be nice to unify the representation. */ in the same
```

Comment:
This makes `Parameter` reusable for `fields` and in general it's a more compact representation. `Parameter` also is just a name with type, so extended types easily fit there, unlike `Method` which has a lot of fields.